plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -24,6 +24,12 @@ There are two ways to deploy Foreman with remote databases:
     * restore the DBs on remote servers
     * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
 
+### Important note: Katello 3.15 changes the default database for Pulp from PostgreSQL v9.x to PostgreSQL v12. External databases should be upgraded to avoid conflicts.
```

Comment:
The entire text block being a larger font looks a little strange.  Maybe instead there could be an "Important notes" header with the text as normal below?

Reply:
@ianballou how about:
![Screenshot from 2020-10-19 08-57-26](https://user-images.githubusercontent.com/266755/96453994-3dfd6180-11e9-11eb-93bc-ab8da590230b.png)